### PR TITLE
Show progress on plugin activation

### PR DESCRIPTION
**Pull request by amiramw**
Fixes: https://github.com/eclipse-theia/theia/issues/6550

Signed-off-by: Amiram Wingarten <amiram.wingarten@sap.com>

#### What it does
Show progress loading message on theia shell status bar when vscode extension is activated. This can happen on activation event or on reconnects.

#### How to test
- On theia start all extensions with activation event * should load with "Loading <extension display name>" message in the status bar.
- Extension with activation event which is not * should show the same message when actual activation occur. For example install https://marketplace.visualstudio.com/items?itemName=camel-tooling.yo and when calling command "yo" there should be a status bar message "Loading yo"
- On client disconnect from the server (for example server restart) reload of extensions should cause also the status bar messages.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/plugin-ext/src/plugin/plugin-manager.ts [hunk 1]
```diff
@@ -185,7 +188,10 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         this.registry.set(plugin.model.id, plugin);
         if (plugin.pluginPath && Array.isArray(plugin.rawModel.activationEvents)) {
             const activation = async () => {
+                const title = `Loading ${plugin.model.displayName || plugin.model.name}`;
+                const id = await this.notificationMain.$startProgress({title, location: 'window'});
```
Comment (akosyakov) [inline packages/plugin-ext/src/plugin/plugin-manager.ts]: please use Theia or VS Code that formatting on save works

File: packages/plugin-ext/src/plugin/plugin-manager.ts [hunk 2]
```diff
@@ -185,7 +188,10 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         this.registry.set(plugin.model.id, plugin);
         if (plugin.pluginPath && Array.isArray(plugin.rawModel.activationEvents)) {
             const activation = async () => {
+                const title = `Loading ${plugin.model.displayName || plugin.model.name}`;
```
Comment (akosyakov) [inline packages/plugin-ext/src/plugin/plugin-manager.ts]: VS code says: `Activating Extensions...`. What loading means btw for a user? It should be at least `Activating`

File: packages/plugin-ext/src/plugin/plugin-manager.ts (hunk 1 above)
Comment (amiramw) [reply packages/plugin-ext/src/plugin/plugin-manager.ts]: fixed

File: packages/plugin-ext/src/plugin/plugin-manager.ts (hunk 2 above)
Comment (amiramw) [reply packages/plugin-ext/src/plugin/plugin-manager.ts]: changed to "Activating"

**Pull request merged**